src/routing_node.rs

```diff
@@ -836,6 +863,139 @@ impl RoutingNode {
                                   refresh_our_own_group: bool) -> RoutingResult {
         unimplemented!()
     }
+
+    fn handle_cache_get(&mut self, request : ExternalRequest) -> Option<Data> {
+        match request {
+            ExternalRequest::Get(data_request) => {
+                match data_request {
+                    DataRequest::StructuredData(data_name, value) => {
+                        match self.structured_data_cache.as_mut() {
```

Comment:
I just looked it up, yes it is, as LruCache updates itself on get; so looks good !